_src/static/sass/base/_base.scss

```diff
@@ -0,0 +1,72 @@
+// ----------------------------
```

Comment:
NIT: Please sync up this title to match our table of contents formatting in https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#table-of-contents